### PR TITLE
Add system integration docs

### PR DESCRIPTION
**Pull request by donadigo**
Adds system integration docs. Please review the grammar of this, I'm not a native speaker.

### CODE REVIEW

File: docs/code/getting-started.md [hunk 1]
```diff
@@ -780,4 +780,163 @@ Let's review what all we've learned:
...
+  )
+  ```
+
+  Rebuild your project and you should good be go.
+
+## Using the Launcher API {#using-launcher-api}
+Once you've set up `libunity` in your build system it's time to write some code.
+
+The first thing you'll need to use the API is your application desktop ID.
+This is usually the filename of your application entry that is installed in `/usr/share/applications`
```
Comment (btkostner) [inline docs/code/getting-started.md]: installed in the `/usr/share/applications` directory like

File: docs/code/getting-started.md [hunk 2]
```diff
@@ -780,4 +780,163 @@ Let's review what all we've learned:
...
+
+This entry instance allows you to modify your entry so that it shows additional information e.g: on the dock. It is up to you, where in the code you want to retrieve this entry, the function is static so there is no problem accessing it. Usually it's your application or main window class.
+
+Showing a `12` number in the badge is as easy as:
+```vala
+entry.count_visible = true;
+entry.count = 12;
+```
+Note that, you have to set the `count_visible` property to `true` when showing a badge counter. 
+Notice the `int64` type of the `count` property. You cannot show real numbers in the badge itself.
```
Comment (btkostner) [inline docs/code/getting-started.md]: The `Note` followed by `Notice` seems kinda weird. Maybe try something like:

> Keep in mind you have to set the `count_visible` property to `true`, and use an `int64` type for the `count`property.

File: docs/code/getting-started.md [hunk 3]
```diff
@@ -780,4 +780,163 @@ Let's review what all we've learned:
...
+
+// Finally, tell libunity to show the desired quicklist
+entry.quicklist = quicklist;
+```
+Please see the [Dbusmenu.Menuitem API](https://valadoc.org/dbusmenu-glib-0.4/Dbusmenu.Menuitem.html) for more details and features.
+
+## Static Quicklists {#static-quicklists}
+The main difference between dynamic and static quicklists is that static ones cannot be changed at runtime. Static quicklists do not involve writing any code or using any external dependencies.
+
+Static quicklists are stored within your `.desktop` file. Those are so called "actions".
```
Comment (btkostner) [inline docs/code/getting-started.md]: `Those` should be `These`

**Comment by donadigo** [comment]
@btkostner @danrabbit fixed.

**Pull request merged**